### PR TITLE
Update, Delete, Aggregate

### PR DESCRIPTION
**Pull request by Akorede3133**
Screenshot (207)
Screenshot (206)
Screenshot (203)
Screenshot (202)
Screenshot (201)
Screenshot (200)
Screenshot (199)
Screenshot (198)
Screenshot (197)
Screenshot (196)
Screenshot (195)
Screenshot (194)
Screenshot (193)
Screenshot (192)
Screenshot (191)
Screenshot (190)
Screenshot (189)
Screenshot (188)
Screenshot (187)
Screenshot (186)
Screenshot (185)
Screenshot (184)
Screenshot (183)
Screenshot (182)
Screenshot (181)
Screenshot (180)
Screenshot (179)
Screenshot (177)
In this PR, the folowing were affected;

* Added a column species of type string to your animals table. Modify your schema.sql file.
* Inserted new sets of data
* Created TRANSACTIONS to update and delete from table
* Wrote queries for aggregate functions

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -14,4 +14,126 @@ SELECT * FROM animals WHERE neutered = TRUE; --Find all animals that are neutere
...
+SELECT COUNT(*) FROM animals
+WHERE escape_attempts = 0;
+
+-- What is the average weight of animals?
+SELECT AVG(weight_kg) FROM animals;
+
+-- Who escapes the most, neutered or not neutered animals?
+
+-- Sum of escape attempts by neutered
+SELECT neutered, SUM(neutered) FROM animals
```
Comment (usorfaitheloho) [inline queries.sql]: You did well, kindly note that the who escapes the most, neutered and non-neutered should be  AVG(escape_attempts) not  SUM(neutered) 😄

File: queries.sql [hunk 2]
```diff
@@ -14,4 +14,126 @@ SELECT * FROM animals WHERE neutered = TRUE; --Find all animals that are neutere
...
+GROUP BY neutered
+HAVING neutered = TRUE;
+
+-- Sum of escape attempts by not neutered
+SELECT neutered, SUM(neutered) FROM animals
+GROUP BY neutered
+HAVING neutered = FALSE;
+
+-- What is the minimum and maximum weight of each type of animal?
```
Comment (usorfaitheloho) [inline queries.sql:112]: Please kindly add a query for this condition, you can use `sql
SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals
   GROUP BY species;
`

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Akorede3133  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Created schema file ✔️
> * Inserted datas into the table ✔️
> * Add more queries ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @Akorede3133 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Implemented the change requested to fix error in the queries and add the missing query.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**